### DISCUSSION TITLE
otPlat APIs

### DISCUSSION THREAD

**Discussion by romacdon** [openthread/openthread#9967] [Q&A] [answered]
Are the various otPlat functions only intended on being called from the OT stack or are these considered APIs that can be called from the application as well?

**Comment by jwhui** [comment]
Platform APIs are generally only intended to be called by OT.

**Reply by romacdon** [reply]
Thanks @jwhui - but are there exceptions? For example, otPlatRadioGetRssi, that may be a function that applications would want to call , right?

**Reply by jwhui** [reply, marked as answer]
Any capabilities beyond what OT requires is platform dependent. For example, a platform could implement reentrant `otPlat` APIs allowing for multiple concurrent invocations.

**Reply by romacdon** [reply]
Thanks @jwhui
